docs/Upgrade-Guide/upgrade_to_8.md

```diff
@@ -0,0 +1,42 @@
+# Upgrade procedure to Gluster 8, from Gluster 7.x, 6.x, 5.x, 4.1.x, and 3.12.x
+
+We recommend reading the [release notes for 8.0](../release-notes/8.0.md) to be
+aware of the features and fixes provided with the release.
+
+> **NOTE:** Upgrade procedure remains the same as with 4.1.x release
+
+Refer, to the [Upgrading to 4.1](./upgrade_to_4.1.md) guide and follow
```

Comment:
As the link we are pointing to is already outdated and will not even be supported. it's better to replace this doc with the newer version. If it's a lot of work, this is fine for now.